en/admin/admin_utils.rst

```diff
@@ -3299,6 +3300,49 @@ The following table shows [options] available with the cubrid tde utility.
 
     This option specifies the password of the DBA.
 
+.. _vacuumdb:
+
+vacuumdb
+--------
+
+The **cubrid vacuumdb** utility is used to vacuum deleted records or unnecessary mvcc related information from records in the database. And it is also used to get the status information about the vacuum. ::
+
+    cubrid vacuumdb [options] database_name
+
+*   **cubrid**: An integrated utility for the CUBRID service and database management.
+
+*   **vacuumdb**: A utility that vacuum deleted records or unnecessary mvcc related information from records in the database.
+
+*   *database_name*: The name of the database to vacuum.
+
+The following shows [options] available with the **cubrid vacuumdb** utility.
+
+.. program:: vacuumdb
+
+.. option:: -o, --output-file=FILE
+
+    This option is used to store the status information about the vacuum in a specified file. The file is created in the current directory. If the **-o** option is not specified, the message is displayed on a console screen. ::
+
+        cubrid vacuumdb -o db_output demodb
+
+.. option:: --dump
+
+    This option is used to display the status information about the vacuum. It currently displays the first log page ID referenced by the vacuum and the log volume name in which it resides. It can be used with both the standalone mode and the client/server mode. ::
```

Comment:
1. I think users don't know what the first log page ID means and why this is important. It seems better to add some elaboration on this.
2. All options can be run in both modes in default, so I don't think the last sentence is needed.

Reply:
1. OK.
2. I wrote that the CS-mode does not currently support vacuuming. However, since the --dump option supports the CS-mode, it is intended to accurately inform a user of this. Still, do you think this phrase is unnecessary? If it is, then I will remove it.

Reply:
Oh, sorry, I misunderstood. You're right.